gov2/README.md

```diff
@@ -1,31 +1,15 @@
-# AWS SDK for Go (v2) code examples
+# Amazon CloudWatch examples for the AWS SDK for Go (v2).
```

Comment:
This should be the language-level README, not the service-level CloudWatch README.